staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -269,6 +271,10 @@ func (o *MirrorOptions) Run() error {
 		}
 	}
 
+	var currentRepo string
```

Comment:
style: inline this variable into the map assignment directly (unless you use it in two places)